CMakeLists.txt

```diff
@@ -125,6 +125,8 @@ option(Mapper_AUTORUN_SYSTEM_TESTS "Run the system tests as part of the Mapper_T
 option(Mapper_AUTORUN_MANUAL_TESTS "Run the system tests as part of the Mapper_Test target" OFF)
 mark_as_advanced(Mapper_AUTORUN_SYSTEM_TESTS Mapper_AUTORUN_MANUAL_TESTS)
 
+option(Mapper_COVERAGE_BUILD "Builds with coverage support" ${Mapper_DEVELOPMENT_BUILD})
```

Comment:
Does it come without negative side effects? I wonder how the coverage instrumentation affects the debugging experience.